components/icu4x_ecma402/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "icu4x_ecma402"
```

Comment:
Do we want to release it under `icu4x_ecma402` name vs. `icu_ecma402` like other components? 
@sffc @mihnita - thoughts?